module_staff.rb

```diff
@@ -0,0 +1,27 @@
+module Staff

```

Comment:
In object oriented programming, we tend to prefer using inheritance over modules.  This way will certainly work, but we generally only use modules for things that aren't central to the object they live in, or which don't make sense in an existing superclass (and it doesn't make sense to add in a middle layer superclass).  In this case, these behaviors are very central to what it means to be a teacher at dbc, so they should be part of a superclass for apprentice and senior teacher, not a module.